README.md

```diff
@@ -27,14 +27,16 @@ Add the module requirement in your `tiapp.xml` file.
 </modules>
 ```
 
-In order to use this module with Titanium you will need to disable app thinning by editing the adding the following property in your `tiapp.xml`
+In order to use this module with Titanium you will need to disable app thinning by editing the adding the following property in your `tiapp.xml`:
 
 ```xml
 <ios>
     <use-app-thinning>false</use-app-thinning>
 </ios>
 ```
 
+Alternatively, you can pass the hashed name of the icon to the methods. You can look up the generated names in `build/iphone/Assets.xcassets`.
```

Comment:
Have you tested this? I couldn't make it work since the hash is generated after the compile and the name of the icons has to be added on the tiapp.xml.